notable/notable.nuspec

```diff
@@ -6,20 +6,42 @@
     <version>1.8.4</version>
     <authors>Fabio Spampinato</authors>
     <owners>Fabio Spampinato, Bradlee Speice</owners>
-    <summary>The markdown-based note-taking app that doesn't suck.</summary>
-    <description><![CDATA[The markdown-based note-taking app that doesn't suck.
+    <summary>The Markdown-based note-taking app that doesn't suck.</summary>
+    <description><![CDATA[## Markdown-Based
```

Comment:
The `<description>` section gets re-written with whatever is in the `Readme.md` when updating; please make sure that the readme is updated as well.